modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+```
+
+[[auth-enable]]
+=== Authentication enabled
+
+Ensure that authentication is set up (or other mitigations are in place). Without SASL authentication enabled, anybody can potentially connect to the Redpanda brokers.
+
+Also ensure that the Admin API is secured, and any users defined in the superusers configuration are appropriately protected with strong credentials.
+
+Example:
+```bash
+rpk cluster config get kafka_enable_authorization
+```
+
+Output:
+```bash
+true
+```
+
+[[tls-enabled]]
+=== TLS Enabled 
+
+Ensure that all public interfaces have TLS enabled.
+
+Example:
+```bash
+grep tls /etc/redpanda/redpanda.yaml
+    kafka_api_tls:
+    admin_api_tls:
+    rpc_server_tls:
+        tls:
+        tls:
+    schema_registry_api_tls:
+```
+
+[[redpanda-tuners]]
+=== Redpanda Tuners
+
+Ensure that tuners have been run on all cluster hosts. Redpanda tuners ensure that the operating system is configured for optimal performance and can have a significant impact on the latency and throughput. In Kubernetes, you may need to run the tuners on the hosts themselves, rather than in containers.
+
+Example:
+```bash
+systemctl status redpanda-tuner
+```
+
+Output:
+```bash
+● redpanda-tuner.service - Redpanda Tuner
+     Loaded: loaded (/lib/systemd/system/redpanda-tuner.service; enabled; vendor preset: enabled)
+     Active: active (exited) since Mon 2024-03-25 12:03:51 UTC; 48min ago
+    Process: 3795 ExecStart=/usr/bin/rpk redpanda tune all $CPUSET (code=exited, status=0/SUCCESS)
+   Main PID: 3795 (code=exited, status=0/SUCCESS)
+
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: cpu                    true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: disk_irq               true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: disk_nomerges          true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: disk_scheduler         true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: disk_write_cache       false    true     false      Disk write cache tuner is only supported in GCP
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: fstrim                 false    false    true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: net                    true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: swappiness             true     true     true
+Mar 25 12:03:51 ip-172-31-10-199 rpk[3795]: transparent_hugepages  false    false    true
+Mar 25 12:03:51 ip-172-31-10-199 systemd[1]: Finished Redpanda Tuner.
+```
+
+Ensure that xref:reference:rpk/rpk-iotune.adoc[`rpk iotune run`] has been run on all hosts. Ensure that the mountpoint listed in this configuration file matches the mountpoint for Redpanda's data directory, usually configured at `/var/lib/redpanda`. See xref:deploy/deployment-option/self-hosted/manual/production/production-deployment.adoc#generate-optimal-io-configuration-settings[Generate optimal I/O configuration settings].
```

Comment:
this is missing a header. this was step 11 in the original doc, so we need to convert to an appropriate section here I guess.

Reply:
I didn't put this on a separate step on purpose. To me it seems like yet another tuner. 
Do we need the separate header anyway?